Remotion/Web/Development.WebTesting/Utilities/RetryUntilTimeout.cs

```diff
@@ -55,27 +55,27 @@ public void Run ()
       _retryUntilTimeout.Run();
     }
 
-    public static void Run ([NotNull] Action action)
+    public static void Run ([NotNull] Action action, TimeSpan timeout = default, TimeSpan retryInterval = default)
     {
       ArgumentUtility.CheckNotNull ("action", action);
       var configuration = new WebTestConfigurationFactory ().CreateDriverConfiguration();
 
       var retryUntilTimeout = new RetryUntilTimeout (
           action,
-          configuration.SearchTimeout,
-          configuration.RetryInterval);
+          timeout == default ? configuration.SearchTimeout : timeout,
+          retryInterval == default ? configuration.RetryInterval : retryInterval);

```

Comment:
I was hesitant to use the ternary conditional operator in this instance as the `configuration` variable is created in any case, even if both `timeout` and `retryInterval` are `default`. One possible alternative would be introducing a private method encapsulating the case distinction, something like this:
```c#
private Tuple<TimeSpan, TimeSpan> GetTimeoutAndRetryInterval(TimeSpan timeout, TimeSpan retryInterval)
    {
      if (timeout != default && timeout != default)
        return Tuple.Create (timeout, retryInterval);

      var configuration = new WebTestConfigurationFactory().CreateDriverConfiguration();
      return Tuple.Create (timeout == default ? configuration.SearchTimeout : timeout, retryInterval == default ? configuration.RetryInterval : retryInterval);
    }
```

Although, in my opinion, we would not gain much by doing so, except that things more would become more verbose than necessary. Hence, I went with the approach in the PR.

Reply:
Using `TimeSpan?` as mentioned above makes this easier, you could just write `timeout ?? configuration.SearchTimeout` or even `timeout ??= configuration.SearchTimeout;` as separate statement.

Concerning the above, imo the ternary operator is a better choice here. It's just so much more code + tuples to extract it into a separate method. But that's just my opinion.

Reply:
Yes, that makes sense. I guess I took the ticket's wording a bit too literally ("(...) should be configurable (...) via default `TimeSpan` parameters).